cmake/FindOpenSyclConfig.cmake

```diff
@@ -0,0 +1,45 @@
+include(FindPackageHandleStandardArgs)
+include(CheckCXXSourceRuns)
+include(CMakePushCheckState)
+
+cmake_push_check_state()
+include(CheckCXXSourceCompiles)
+
+if (NOT TARGET opensycl-config::interface)
+  if (NOT DEFINED DEVICE_ARCH)
+    message(FATAL_ERROR "`DEVICE_ARCH` env. variable is not provided.")
+  else()
+
+  add_library(opensycl-config::interface INTERFACE IMPORTED)
+ 
+  execute_process(COMMAND python3 ${CMAKE_CURRENT_LIST_DIR}/find-opensycl.py -i
```

Comment:
Just as a suggestion: look for Python 3 as a package, if we're using it... Just in case?

(cf. also https://cliutils.gitlab.io/modern-cmake/chapters/basics/programs.html )